### ISSUE TITLE
Electron icon shown instead of the ones in /build

### ISSUE THREAD

**Issue by alexmozzhakov** [ifedapoolarewaju/igdm#360]

**Comment by ifedapoolarewaju** [comment]
Hi, could you please add some description to this? What version are you using? On what OS?

**Comment by alexmozzhakov** [comment]
macOS, git’s master head

**Comment by alexmozzhakov** [comment]
I will send you uname output when I get home

**Comment by ifedapoolarewaju** [comment]
> macOS, git’s master head

so you are running it in dev mode? With `npm start`?

**Comment by alexmozzhakov** [comment]
@ifedapoolarewaju yes, the output was
```
> IGdm@2.5.1 start /Users/alex/Downloads/igdm
> gulp build && electron .

[09:51:22] Using gulpfile ~/Downloads/igdm/gulpfile.js
[09:51:23] Starting 'build'...
[09:51:23] Starting 'html'...
[09:51:23] Finished 'html' after 326 ms
[09:51:23] Finished 'build' after 330 ms
Checking for update
Update for version 2.5.1 is not available (latest version: 2.5.1, downgrade is disallowed).
2018-06-17 09:51:45.965 Electron Helper[27542:2261027] Couldn't set selectedTextBackgroundColor from default ()
```
uname -a output:
`Darwin Macbook.local 16.7.0 Darwin Kernel Version 16.7.0: Tue Jan 30 11:27:06 PST 2018; root:xnu-3789.73.11~1/RELEASE_X86_64 x86_64`

**Comment by alexmozzhakov** [comment]
OK, not an issue https://stackoverflow.com/a/31538436/5182070
